packages/dbml-core/src/parse/postgresql/Base_rules.pegjs

```diff
@@ -59,6 +59,12 @@ data_type "VALID TYPE" = c1:"CHARACTER"i _ c2:"VARYING"i _ args:("("expression")
     args: args ? args[1] : null
   }
 }
+/ c1:("INTEGER"i/"INT"i/"SMALLINT"i/"BIGINT"i) _ c2:"GENERATED"i _ c3:"BY"i _ c4:"DEFAULT"i _ c5:"AS"i _ c6:"IDENTITY"i {
```

Comment:
We already have `column_constraint` rules in `Create_table_normal.pegjs` that defined `GENERATED BY DEFAULT AS IDENTITY`. Should use that instead of include this in type name.